src/generateChangelog.js

```diff
@@ -28,7 +28,14 @@ async function fetchAllReleases(repoInfo) {
     stillGettingReleases = data.length === perPage;
     page += 1;
   }
-  return results.filter(_ => /^\d/.test(_.name));
```

Comment:
I think this should be the default regex if none is provided

Reply:
Scratch that, @pe4cey had a better idea. Let's support both these formats by default `3.2.1` and `v3.2.1`.

Reply:
Done @oskarhane, Thank you for all the comments!